app/admin/RTMediaFormHandler.php

```diff
@@ -57,39 +64,44 @@ public static function textarea( $args, $echo = true ) {
 			'show_desc' => false,
 		);
 		$args     = wp_parse_args( $args, $defaults );
-		extract( $args );
 
-		if ( ! isset( $value ) ) {
-			trigger_error( esc_html__( 'Please provide a "value" in the argument.', 'buddypress-media' ) );
+		if ( ! isset( $args['value'] ) ) {
+			trigger_error( esc_html__( 'Please provide a "value" in the argument.', 'buddypress-media' ) ); // @codingStandardsIgnoreLine
```

Comment:
Use `phpcs:ignore` instead of `@codingStandardsIgnoreLine` because its deprecated.

Reply:
Address this at other places in this file.